install-panel.sh

```diff
@@ -665,21 +716,18 @@ function letsencrypt {
 
   # Install certbot
   if [ "$OS" == "debian" ] || [ "$OS" == "ubuntu" ]; then
-    apt-get install certbot -y
+    apt-get -y install certbot python-certbot-nginx
   elif [ "$OS" == "centos" ]; then
-    [ "$OS_VER_MAJOR" == "7" ] && yum install certbot
-    [ "$OS_VER_MAJOR" == "8" ] && dnf install certbot
+    [ "$OS_VER_MAJOR" == "7" ] && yum -y -q install certbot python-certbot-nginx
+    [ "$OS_VER_MAJOR" == "8" ] && dnf -y -q install certbot python-certbot-nginx
   else
     # exit
     print_error "OS not supported."
     exit 1
   fi
 
-  # Stop nginx
-  systemctl stop nginx
-
   # Obtain certificate
-  certbot certonly --no-eff-email --email "$email" --standalone -d "$FQDN" || FAILED=true
+  certbot certonly --no-eff-email --email "$email" --nginx -d "$FQDN" || FAILED=true
```

Comment:
Does this work with auto-renewal or does it have to spin up a temporary web server (therefore being blocked since nginx is running)?

Reply:
Updated, should work now.

Reply:
Tested. Auto-renewal is working. Had to change quite a few things to get it working, but I got it working.
It doesn't spin up any temporary servers, because it is using `nginx` to do it all.